### PR TITLE
image: error out on duplicate tar entry

### PR DESCRIPTION
**Pull request by runcom**
Should address the code change requested in https://github.com/opencontainers/image-spec/issues/196

/cc @philips @stevvooe @vbatts @s-urbaniak 

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by philips** [comment]
LGTM

**Comment by runcom** [comment]
network error in Travis, any way to re-start it?

**Comment by philips** [comment]
@runcom add whitespace to your commit message with `git commit --amend` and repush...

**Comment by jonboulle** [comment]
@caniszczyk could we get maintainers access to Travis to be able to restart builds?

File: image/manifest.go [hunk 1]
```diff
@@ -154,6 +155,10 @@ loop:
 			}
 		}
 		path := filepath.Join(dest, hdr.Name)
+		if entries[path] {
+			return fmt.Errorf("%s has been already processed", path)
```
Comment (jonboulle) [inline image/manifest.go]: IMHO "duplicate entry for %s"  would be better

**Comment by runcom** [comment]
tests now green though

File: image/manifest.go (hunk 1 above)
Comment (runcom) [inline image/manifest.go]: fixing

Comment (runcom) [inline image/manifest.go]: Fixed

**Comment by jonboulle** [comment]
lgtm

**Comment by vbatts** [comment]
LGTM

**Comment by runcom** [comment]
@wking I have a branch which will take care of what you're saying but I'm buried with other stuff/PRs that are higher priority to me right now.

**Comment by runcom** [comment]
Can you open an issue to track this refactor of the unpack code?

**Comment by wking** [comment]
On Thu, Sep 01, 2016 at 09:51:16AM -0700, Antonio Murdaca wrote:
> Can you open an issue to track this refactor of the unpack code? 

I've opened #242 to track cleanup on unpack errors.

**Pull request merged**